### ISSUE TITLE
Instance: The timestamp links (within chat messages) have non-descriptive accessible names | WCAG 2.2 Violation - There are links with non-descriptive accessible names present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#35]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1. 

### Screenshot

![Screenshot of an example of one of the timestamp links with a non-descriptive accessible name ('8 hours ago').](https://github.com/user-attachments/assets/63704002-456a-4d62-bd9b-356cc92bff2c)

### Code Reference

```
<!-- An example of one of the timestamp links with a non-descriptive accessible name ("8 hours ago") -->
<span class="chat-timestamp">
	<a href="https://ubccms.test/#comment-34">8 hours ago</a>
</span>
```
